behat.yml.dist

```diff
@@ -45,6 +45,9 @@ default:
                 - Ergonode\Core\Test\Behat\Context\ApiAuthContext:
                     JWTTokenManager: '@lexik_jwt_authentication.jwt_manager'
                     request: '@Behatch\HttpCall\Request'
+                - Ergonode\Core\Test\Behat\Context\ApiAuthTokenContext:
+                    request: '@Behatch\HttpCall\Request'
+                    authHeader: 'JWTAuthorization'
```

Comment:
Perhaps set it as the container parameter and the value can be easily shared and set in just one place?